onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -260,9 +260,16 @@ void RegisterConvActivationFusionRules(SelectorActionRegistry& registry) {
   const auto name = "ConvAct";
   auto action = std::make_unique<actions::FuseConvActivationAction>();
 #if !defined(ORT_MINIMAL_BUILD)
+  SelectorActionRegistry::OpVersionsMap op_versions_map;
+  std::string kMSInternalNHWCDomainConv = std::string(kMSInternalNHWCDomain) + ":Conv";
+  std::string kMSInternalNHWCDomainConvTranspose = std::string(kMSInternalNHWCDomain) + ":ConvTranspose";
```

Comment:
nit: the 'k' prefix implies const per the [style guide](https://google.github.io/styleguide/cppguide.html#Constant_Names) so these should be const

Reply:
removed the first letter k.